portal/server/generateApp.js

```diff
@@ -124,6 +124,7 @@ const generateApp = () => {
   // error handler
   app.use((error, req, res, next) => {
     if (error.code === 'EBADCSRFTOKEN') {
+      console.error('The user\'s CSRF token is incorrect, redirecting the user to /.');
```

Comment:
this reads slightly odd, happy to keep as is, but could also use `PORTAL_API_URL`?